multimodal/src/autogluon/multimodal/utils/save.py

```diff
@@ -112,3 +114,37 @@ def process_save_path(path, resume: Optional[bool] = False, raise_if_exist: Opti
             path = None
 
     return path
+
+
+def setup_save_path(
+    resume: Optional[bool] = None,
+    old_save_path: Optional[str] = None,
+    proposed_save_path: Optional[str] = None,
+    hyperparameter_tune_kwargs: Optional[dict] = None,
+    warn_if_exist: Optional[bool] = True,
+    num_gpus: Optional[int] = None,
+):
+    save_path = None
+    raise_if_exist_for_proposed_path = True
+    if num_gpus and num_gpus > 1:
+        raise_if_exist_for_proposed_path = (
+            False  # TODO: Here we allow overwrite previous save_path if we are using multi gpu
+        )
```

Comment:
If users give a `proposed_save_path`, which already exists, and use multi gpus, this will create a new folder. If users try to use their specified path, they would still find the old results there. I think this behavior can cause user confusions.

Reply:
For now, adding a TODO item is fine.